Meta/CMake/code_generators.cmake

```diff
@@ -2,7 +2,7 @@
 # Functions for generating sources using host tools
 #
 
-function(compile_gml source output string_name)
+function(stringify_gml source output string_name)
```

Comment:
Maybe do the `compile_gml` -> `stringify_gml` change in a separate commit, makes it less jarring to look at the scroll bar while reviewing. :^)